package.json

```diff
@@ -12,22 +12,24 @@
     "cordova": {
         "plugins": {
             "cordova-plugin-whitelist": {},
+            "@bluedot-innovation/cordova-plugin": {},
             "cordova.plugins.diagnostic": {
-                "ANDROIDX_VERSION": "1.+"
-            },
-            "@bluedot-innovation/cordova-plugin": {}
+                "ANDROIDX_VERSION": "1.0.0",
+                "ANDROIDX_APPCOMPAT_VERSION": "1.3.1"
+            }
         },
         "platforms": [
             "ios",
             "android"
         ]
     },
     "devDependencies": {
-        "@bluedot-innovation/cordova-plugin": "^3.0.0",
+        "@bluedot-innovation/cordova-plugin": "file:../PointSDK-Plugin-Cordova",
```

Comment:
Be sure to revert this hard coded path before merging.

Reply:
Updated the dependency path with the Cordova plugin git-branch